stardog/values.yaml

```diff
@@ -83,3 +83,13 @@ zookeeper:
     # serverPasswords: secretPassword,
   metrics:
     enabled: true
+  affinity:
+    podAntiAffinity:
+      requiredDuringSchedulingIgnoredDuringExecution:
+        - labelSelector:
+            matchExpressions:
```

Comment:
We could add `{{ .Release.Name }}` for the `release`label in order to allow multiple zookeepers in the same namespace.

Reply:
Good catch but it's unfortunately not possible to use variables in the `values.yaml` file.

Reply:
But I found another issue.

Reply:
Ok, it's solved.